src/export/digitalPlanning/model.ts

```diff
@@ -495,21 +511,22 @@ export class DigitalPlanning {
     }
   }
 
-  private getProposal(): Payload["data"]["proposal"] {
+  private getProposal(): Proposal {
     const baseProposal = {
-      projectType: this.passport.data?.["proposal.projectType"].map(
-        (project: string) => {
-          return {
-            value: project,
-            description: this.findDescriptionFromValue("ProjectType", project),
-          };
-        },
-      ),
+      projectType: (
+        this.passport.generic<string[]>("proposal.projectType") || []
+      ).map((project: string) => {
+        return {
+          value: project,
+          description: this.findDescriptionFromValue("ProjectType", project),
+        } as ProjectType;
+      }),
       description:
         this.passport.data?.["proposal.description"] || "Not provided",
+      boundary: this.getBoundary(),
       date: {
-        start: this.passport.data?.["proposal.start.date"],
-        completion: this.passport.data?.["proposal.completion.date"], // this.passport.data?.["proposal.finish.date"],
+        start: this.passport.generic<string>("proposal.start.date")!,
+        completion: this.passport.generic<string>("proposal.completion.date")!, // this.passport.data?.["proposal.finish.date"],
```

Comment:
I'm a little fuzzy on this TS syntax - are the `!` indicating required/always expected here? start & completion dates are optional per the schema, and are not expected to be provided throughout

Reply:
Good spot! This may have been something which was required on my first pass through here.